docs/configure/typescript.md

```diff
@@ -90,7 +90,7 @@ Now, when you define a story or update an existing one, you'll automatically get
 
 ### The `satisfies` operator is not working as expected
 
-Out of the box, Storybook supports the `satisfies` operator for almost every framework already using TypeScript version 4.9 or higher. However, due to the constraints of the Angular and Web Components framework, you might run into issues when applying this operator for additional type safety. This is primarily due to how both frameworks are currently implemented, making it almost impossible for Storybook to determine if the component property is required. If you encounter this issue, we recommend reaching out to the maintainers using the default communication channels (e.g., [Discord server](https://discord.com/channels/486522875931656193/570426522528382976), [GitHub discussion](https://github.com/storybookjs/storybook/discussions/20988)).
+Out of the box, Storybook supports the `satisfies` operator for almost every framework already using TypeScript version 4.9 or higher. However, due to the constraints of the Angular and Web Components framework, you might run into issues when applying this operator for additional type safety. This is primarily due to how both frameworks are currently implemented, making it almost impossible for Storybook to determine if the component property is required. If you encounter this issue, we recommend reaching out to the maintainers using the default communication channels (e.g., [Discord's Angular Working Group channel](https://discord.com/channels/486522875931656193/855364585632038913), [Web Components Working Group channel](https://discord.com/channels/486522875931656193/855354327854350346), [GitHub discussions](https://github.com/storybookjs/storybook/discussions/20988)).
```

Comment:
@joevaugh4n and @vanessayuenn It seems that we still have those channels available despite being archived, and we should retask them as a means to engage with maintainers and have a communication pipeline to address features such as this one.

Let me know, and we'll take it from there.

Reply:
I don't think the working group channels are archived, though not super active. However, I think the working group channels should be for existing and potential contributors to discuss domain specific things, and not for offering support. This section is under "troubleshooting" so directing folks to "help" in discussion is more appropriate. What do you think?

Reply:
@vanessayuenn I left the mention to those channels as it's the only place where we can follow up with anyone willing to help us with this issue/integration and have a faster point of contact to follow up with the maintainers. 
If you don't mind, I'll wait for @joevaugh4n 's input, and we see where we go from there. Sounds good?

Reply:
Hey, @jonniebigodes! Yes, to clarify, those channels aren't archived. They're still available under the 'working groups' category but are relatively inactive. Like Vanessa says, imo we should refer anyone with Angular support requests to discussions, as those wg channels are mostly for maintainer/contributor usage. Sound good?